addons/binding/org.openhab.binding.keba/ESH-INF/thing/kecontact.xml

```diff
@@ -135,5 +136,11 @@
 		<description>Power consumption in Wh. Total energy consumption is added up after each completed charging session</description>
 		<state pattern="%d Wh" readOnly="true"></state>
 	</channel-type>
+	<channel-type id="display" advanced="true">
```

Comment:
XML files should be indented using tabs according to our code formatter, this looks like the new section is indented with spaces instead.